README.md

```diff
@@ -168,6 +168,23 @@ server.route({
  curl http://localhost:3000/foo/math/add?a=2&b=2
 ```
 
+## Migrate from Hemera to Seneca
```

Comment:
Thanks for making this official, sorry about the delay. I had been focused on rewriting ALL my seneca code to move to hemera.

This should also read:
`Migrate from Seneca to Hemera`

As I'm guessing most people will be leaving Seneca and moving to Hemera.